uqcsbot/minecraft.py

```diff
@@ -48,6 +49,27 @@ async def send_rcon_command(self, command: str):
 
         return response
 
+    @app_commands.command()
+    async def mcplayers(self, interaction: discord.Interaction):
+        """Returns the number and list of people currently playing on the Minecraft server."""
+        server = JavaServer.lookup(
+            "minecraft.uqcs.org:25605"
```

Comment:
Alright I have confirmed this is different from the RCON addr. Probably don't hard-code it - add a new environment variable. You should be able to copy how the other MC commands do it.